docs/getting-started.md

```diff
@@ -138,3 +133,41 @@ provide your own certificates, please refer to [Securing Control Plane](/docs/se
 
 To use antctl, the Antrea command-line tool, please refer to this
 [guide](/docs/antctl.md).
+
+## Features
+
+### Antrea Network Policy
+Besides Kubernetes NetworkPolicy, Antrea also implements its own Network Policy
+CRDs, which provide advanced features including: policy priority, tiering, deny
+action, external entity, and policy statistics. For more information on usage of
+Antrea Network Policies, refer to the [Antrea Network Policy document](antrea-network-policy.md).
+
+### IPsec encryption
+Antrea supports encrypting GRE tunnel traffic with IPsec. To deploy Antrea with
+IPsec encryption enabled, please refer to [this guide](/docs/ipsec-tunnel.md).
+
+### Network flow visibility
+Antrea supports exporting network flow information using IPFIX, and provides a
+reference cookbook on how to visualized the exported network flows using Elastic
+Stack and Kibana dashboards. For more information, refer to the [network flow
+visibility document](network-flow-visibility.md).
+
+### Octant UI
+Antrea ships with an Octant UI plugin which can show runtime information of Antrea
+components and perform Antrea Traceflow operations. Refer to [this guide](octant-plugin-installation.md)
+to learn how to install Octant and the Antrea plugin.
+
+### OVS hardware offload
+Antrea can offload OVS flow processing to the NICs that support OVS kernel
+hardware offload using TC. The hardware offload can improve OVS performance
+significantly. For more information on how to configure OVS offload, refer to
+the [OVS hardware offload guide](ovs-offload.md).
+
+### Prometheus metrics
```

Comment:
Nit: Making the first letter of all words in section names uppercase.
In some section names, this was the case "Antrea Network Policy", but not for all. Is there any reason for this. If not, I think it is better to make this consistent for all section names even in architecture.md.

Reply:
Guess no common convention followed by all docs, but seems this doc does capitalize the first letter in section names. So, changed as you suggested.